arkouda/pdarraycreation.py

```diff
@@ -196,13 +207,23 @@ def array(
 
     # If a is already a pdarray, do nothing
     if isinstance(a, pdarray):
+        # TODO update if max_bits is set
```

Comment:
Just making sure the max bit functionality was not supposed to be included in this PR.

Reply:
Yeah I am planning on doing this in a future PR